packages/bsky/Dockerfile

```diff
@@ -6,12 +6,11 @@ COPY ./*.* ./
 # NOTE bsky's transitive dependencies go here: if that changes, this needs to be updated.
 COPY ./packages/bsky ./packages/bsky
 COPY ./packages/api ./packages/api
-COPY ./packages/aws ./packages/aws
```

Comment:
we might need cloudfront in here for doing invalidations 🤔

Reply:
but i guess we can get that in when we get to blob takedowns in appview

Reply:
Yeah, I think we'll pull it back in once there are codepaths using it.